src/basho_bench.erl

```diff
@@ -70,7 +72,10 @@ main(Args) ->
     application:set_env(lager, crash_log, CrashLog),
     lager:start(),
 
-    application:ensure_all_started(couch),
+    {ok, _} = application:ensure_all_started(couch),
```

Comment:
We need to make this optional. Ideally we don't need it at all, but for the moment we should at least have a way to toggle it otherwise we can't use this tool for any HTTP benchmarks. Perhaps we check an environment variable to start couch and do this conditionally for now.